### DISCUSSION TITLE
power supply sometimes goes into protection

### DISCUSSION THREAD

**Discussion by kotovasia5120** [kachurovskiy/nanoels#236] [Q&A] [answered]
When changing the direction of movement z, the power supply sometimes goes into protection.
I tried to reduce the initial acceleration and put capacitors on the power supply.
What can be done or what power supply to buy?
I use 48v.
On x separate power supply.

**Comment by kachurovskiy** [comment]
Z axis is heavier since it contains X axis within it so it's understandable that it only affects the Z axis.

1. Which stepper driver do you use? Perhaps there's a current setting there.
2. Which capacitors did you try? Perhaps it needs to be larger.
3. Is there a braking resistor support on the driver? Perhaps it's not the acceleration but the rapid braking that's causing the problem?

**Reply by kotovasia5120** [reply]
1. dm542. Tried different current limits
2. 500uF. I also tried connecting two power supplies at the same time.
3. As I understand it, there is no brake.
I think the power supply either does not output the nominal value, or does not have enough capacity for the starting impulse.
https://www.vivaleclighting.com/product/magnetic-track-light-200w-48v-power-supply
When changing direction, the indicator on the power supply goes out for a split second.

**Reply by kachurovskiy** [reply]
I suggest getting a quality driver from a good brand like STEPPERONLINE or RTELLIGENT, looks like your driver is unbranded? Or perhaps doubling the capacitor.

**Reply by kotovasia5120** [reply]
> I suggest getting a quality driver from a good brand like STEPPERONLINE or RTELLIGENT, looks like your driver is unbranded? Or perhaps doubling the capacitor.

Now I connected a power supply from x to 24v 2a and with it the change of direction occurs without problems.
Only after gaining speed does it stop, skipping steps. I will limit the maximum speed
So it seems like you only need to choose a good power supply.
On x.
https://meanwell.by/na-din-rejku/ac-dc-preobrazovatel-hdr-60-24.html

**Reply by kotovasia5120** [reply, marked as answer]
It works well with this one 
![IMG_20250120_224110_485.jpg](https://github.com/user-attachments/assets/c0e9cf0e-b576-4de5-8eaa-8c5cd6a8be8f)
